blocks/editable/index.js

```diff
@@ -570,7 +570,6 @@ export default class Editable extends Component {
 				if ( event.shiftKey || ! this.props.onSplit ) {
 					this.editor.execCommand( 'InsertLineBreak', false, event );
 				} else {
-					event.stopImmediatePropagation();
```

Comment:
Is this necessary? it was added here https://github.com/WordPress/gutenberg/pull/3917 by @mcsf (Trying to see if this introduces any regression)

Reply:
Indeed, I can't spot regressions by removing that line. It should be necessary, per #3867. I wonder if something changed in the stack, e.g. some of these operations happening with deferring?

Reply:
Thanks for finding some related PRs. I'd say that if there's no regressions then we're fine to remove this line. It's necessary for this PR so that `maybeStartTyping()` triggers when ENTER is pressed.